code/game/objects/structures/hybrisa_props.dm

```diff
@@ -2077,9 +2090,8 @@
 	bound_width = 64
 	bound_height = 32
 	density = FALSE
-	health = 1000
 	anchored = TRUE
-	layer = 8
+	layer = FACEHUGGER_LAYER
```

Comment:
Make this its own layer define even if the facehugger layer happens to be the value you want (you can have as many defines as you want with the same resulting value). Should we want to change the layering of a facehugger, it doesn't make sense for this sign to also have its layer changed.